gui/entities/Tile.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <array>
+#include <vector>
+
+namespace gui {
+
+  struct Tile{
+    std::array<int, 7> resources = {0, 0, 0, 0, 0, 0, 0};
```

Comment:
```suggestion
    std::array<int, NBR_OF_RESSOURCES> resources = {0, 0, 0, 0, 0, 0, 0};
```
Évite les magic numbers